### PR TITLE
Fix thread summary padding

### LINKED ISSUE #8837: Thread summaries "Nextcloud assistant" and "Go to newest message" have unbalanced padding
### Steps to reproduce

1. Set up the app including text processing
2. Enable thread summaries
3. Open a long thread
4. Look at the summary

### Expected behavior

Good design

### Actual behavior

Bildschirmfoto vom 2023-09-07 08-16-44

### Mail app version

3.4

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

cc @hamza221

### PR DESCRIPTION
**Pull request by GretaD**
after
Screenshot from 2023-09-08 15-36-44

fixes #8837

### CODE REVIEW

File: src/components/ThreadSummary.vue [hunk 1]
```diff
@@ -23,6 +23,7 @@
 			</div>
 		</div>
 		<div class="summary__body">
+			<LoadingSkeleton v-if="loading" :number-of-lines="1" :with-avatar="false" />
```
Comment (ChristophWurst) [inline src/components/ThreadSummary.vue:26]: two loading skeletons for one summary?

Comment (GretaD) [reply src/components/ThreadSummary.vue]: thats by mistake, of course

**Review by st3iny — APPROVED** [review]
Tested and works.

**Pull request merged**